lib/jinterface/java_src/com/ericsson/otp/erlang/OtpOutputStream.java

```diff
@@ -818,13 +815,10 @@ public void write_ref(final String node, final int[] ids, final int creation) {
 
 	write_atom(node);
 
-	write1(creation & 0x3); // 2 bits
-
-	// first int gets truncated to 18 bits
-	write4BE(ids[0] & 0x3ffff);
+	write4BE(creation);
 
 	// remaining ones are left as is
```

Comment:
This comment should probably be updated as well

Reply:
Yes. Thank you.